### ISSUE TITLE
Prompt not detected - upload failed

### ISSUE THREAD

**Issue by gfwilliams** [espruino/EspruinoWebIDE#194]
The IDE checks for a prompt - but it's not very smart about it... It just checks ~3sec after upload so if upload completes, everything is ok, but then you manually run a function that takes a while to complete it'll still come up and automatically Ctrl-C your code.

It'd be better if it was able to detect that `>` had been sent after upload at any point, and then cancel it.

**Comment by IeiuniumLux** [comment]
@gfwilliams, are there any plans to address this?

**Comment by gfwilliams** [comment]
Yes. Other things have taken priority recently but it's definitely something I want to get fixed.
